TUTORIAL.md

```diff
@@ -2,62 +2,73 @@
 
 `cargo install voidmap`
 
-if you don't have cargo, an easy way to get and manage
-it is [via rustup](https://www.rustup.rs/). Ensure that `~/.cargo/bin`
-is in your `$PATH` afterward, so that you can use the `rustup` and `cargo`
+if you don't have cargo, an easy way to get and manage it is [via
+rustup](https://www.rustup.rs/). Ensure that `~/.cargo/bin` is in your
+`$PATH` afterward, so that you can use the `rustup` and `cargo`
 commands.
 
-If you get errors along the lines of ``error: the `?' operator is not stable`` then
-you need to update your rust compiler. If you installed rust with rustup, this can
-be accomplished with `rustup update`.  Requires a recent stable rust compiler,
-`1.14.0` or higher is recommended. This can be checked with `rustc --version`.
-If you have installed rust with rustup, but you have an old version, there may be
-an older version previously installed on your system. Verify that `which cargo`
-outputs a path that belongs to your `.cargo/bin` directory.
+If you get errors along the lines of
+`` error: the `?' operator is not stable `` then you need to update your
+rust compiler. If you installed rust with rustup, this can be
+accomplished with `rustup update`. Requires a recent stable rust
+compiler, `1.14.0` or higher is recommended. This can be checked with
+`rustc --version`. If you have installed rust with rustup, but you have
+an old version, there may be an older version previously installed on
+your system. Verify that `which cargo` outputs a path that belongs to
+your `.cargo/bin` directory.
 
 #### startup
 
 `void`
 
-this attempts to use `$HOME/.void.db` as a storage file.
-if you'd like to specify a different storage file:
+this attempts to use `$HOME/.void.db` as a storage file. if you'd like
+to specify a different storage file:
 
 `void [/path/to/savefile]`
 
-
 #### basic node creation
-1. C-n # create new node (C-x means Ctrl+x)
-1. type some text to populate it (empty nodes are automatically deleted when unselected)
-1. Tab # create subnode
-1. type some text to populate it
-1. Enter # create sibling node
-1. type text to populate it
+
```

Comment:
Good thinking to add a linebreak here
that makes this markdown render in other engines as well